bootable-jar/WFLY-14934_Server_Mode.adoc

```diff
@@ -0,0 +1,122 @@
+= Server Mode for Bootable JAR Maven Plugin
+:author:           Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Introduce a mode that makes the Bootable JAR Maven Plugin to build a server installation instead of a Bootable JAR.
```

Comment:
maybe: "Introduce a mode in the Bootable JAR Maven Plugin to build a server installation with the application deployment. This mode is an alternative to the existing mode that bundles the server installation and the application deployment inside an executable Bootable JAR."

Reply:
Thanks. Fixed.